srcpkgs/ekushey-fonts-ttf/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'ekushey-fonts-ttf'
+pkgname=ekushey-fonts-ttf
+version=20191020
+revision=1
+archs=noarch
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
```

Comment:
create_wrksrc is not needed with build_style=fetch, remove

Reply:
Removed